Stripe/STPSourceKlarnaDetails.m

```diff
@@ -0,0 +1,48 @@
+//
+//  STPSourceKlarnaDetails.m
+//  Stripe
+//
+//  Created by David Estes on 11/19/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import "STPSourceKlarnaDetails.h"
+
+#import "NSDictionary+Stripe.h"
+
+@interface STPSourceKlarnaDetails()
+@property (nonatomic, copy, readwrite) NSString *clientToken;
```

Comment:
is there anything from our docs or Klarna docs we can use to document this field?

Reply:
Added a link to Klarna's site.